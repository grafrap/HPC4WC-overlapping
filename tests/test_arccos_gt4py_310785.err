slurmstepd: error: *** JOB 310785 ON nid005204 CANCELLED AT 2025-08-15T22:49:30 DUE TO TIME LIMIT ***
