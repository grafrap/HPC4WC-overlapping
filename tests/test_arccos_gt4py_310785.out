Job started on nid005204 -> Running accuracy tests for gt4py
Using Python at: /users/class182/HPC4WC_venv/bin/python
Python 3.12.5
